src/deploy.sh

```diff
@@ -163,13 +160,11 @@ function modules_install()
 #   available kernels in a single line separated by commas. If it gets 0 it
 #   will display each kernel name by line.
 # @target Target can be 1 (VM_TARGET), 2 (LOCAL_TARGET), and 3 (REMOTE_TARGET)
-# @unformatted_remote We expect the REMOTE:PORT string
 function list_installed_kernels()
 {
   local flag="$1"
   local single_line="$2"
   local target="$3"
-  local unformatted_remote="$4"
```

Comment:
At line 437, `list_installed_kernels` is still called with this fourth parameter.